recipes/triqs/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = "2.2.1" %}
+
+package:
+  name: triqs
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/triqs/releases/download/{{ version }}/triqs-{{ version }}.tar.gz
+  sha256: 523d60a98af0c07e972338f3a5f07d7ea1232d6e67b0f063e139d986f09b9864
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+
+requirements:
+  build:
+    - cmake
+    - make
+    - clangxx >=8
```

Comment:
```suggestion
    - clangxx >=8  # [osx]
```

Reply:
and on linux what version of gcc will be used?

Reply:
7.3.0